async_test.go

```diff
@@ -167,12 +182,18 @@ func TestProcessDownloadJobs(t *testing.T) {
 }
 
 func TestAddMongoDBFilesToCacheWithBadPath(t *testing.T) {
+	_, cancel := context.WithTimeout(context.Background(), taskTimeout)
+	defer cancel()
```

Comment:
this doesn't do anything and can be removed.

Reply:
Removed.